tests/table/test_insert.py

```diff
@@ -76,3 +81,23 @@ def test_insert_returning_alias(self):
         )
 
         self.assertListEqual(response, [{"manager_name": "Maz"}])
+
+
+class TestOnConflict(DBTestCase):
```

Comment:
You can use all the tests from my PR (with modification to the new` on_conflict` method) as they cover all possibilities (single column, multiple columns, etc.) for all supported databases.

Reply:
Yes, I'll reuse most of those tests - thanks.